geode-core/src/main/java/org/apache/geode/internal/cache/DiskStoreAttributes.java

```diff
@@ -42,14 +43,20 @@
   public long timeInterval;
 
   public int[] diskDirSizes;
-
+  private DiskDirSizesUnit diskDirSizesUnit;
   public File[] diskDirs;
 
   public String name;
 
   private volatile float diskUsageWarningPct;
   private volatile float diskUsageCriticalPct;
 
+  /**
+   * The default disk directory size unit. Only change it for testing purposes.
+   */
+  @MakeImmutable
+  static DiskDirSizesUnit DEFAULT_DISK_DIR_SIZES_UNIT = DiskDirSizesUnit.MEGABYTES;
```

Comment:
This doesn't actually change, right? I think it could be final and the second sentence from the comment could be removed.